Release 0.0.2 (#31)
pysat/pysatSpaceWeather: README.md

<div align="left">
        <img height="0" width="0px">
        <img width="20%" src="https://raw.githubusercontent.com/pysat/pysatSpaceWeather/main/docs/figures/pysatSpaceWeather.png" alt="pysatSpaceWeather" title="pysatSpaceWeather"</img>
</div>

# pysatSpaceWeather: pysat support for Space Weather Indices
[![Build Status](https://travis-ci.com/pysat/pysatSpaceWeather.svg?branch=main)](https://travis-ci.com/pysat/pysatSpaceWeather)
[![Coverage Status](https://coveralls.io/repos/github/pysat/pysatSpaceWeather/badge.svg?branch=main)](https://coveralls.io/github/pysat/pysatSpaceWeather?branch=main)
[![DOI](https://zenodo.org/badge/287377838.svg)](https://zenodo.org/badge/latestdoi/287377838) [![Documentation](https://readthedocs.org/projects/pysatspaceweather/badge/?version=latest)](https://pysatspaceweather.readthedocs.io/en/latest/?badge=latest)

This module handles solar and geomagnetic indices needed for scientific and
operational projects.

# Installation

> The following instructions will allow you to install pysatSpaceWeather and give

Reviewer: ```suggestion
The following instructions enable (?) you to install pysatSpaceWeather and give
```

Author: Why the `(?)`?

Reviewer: Trying to indicate it was a style suggestion though it didn't work.

Reviewer: Longer explanation. First I noticed the voice was passive, and this is a nit here, plus the instructions don't actually allow folks to install the software, the license does. The first replacement word I came up with was enable, more active, but still suffered from not being fully correct. 
How about 'guide' instead?

Author: Yeah, you have to put the question mark outside of the code suggestion box :P. Ok, I think your nitpick is very small and unimportant... BUT you are technically correct.  And if we know anything, it's that technical correctness is the best kind of correctness.

https://i.redd.it/8sdp12ekctz31.jpg

```
It should be noted that this is a working branch and is subject to change.

# Examples

The instrument modules are portable and designed to be run like any pysat
instrument.

```

> import pysat

Author: ```suggestion
import pysat
import pysatSpaceWeather
```

from pysatSpaceWeather.instruments import sw_dst

dst = pysat.Instrument(inst_module=sw_dst)
```

Another way to use the instruments in an external repository is to register the
instruments.  This only needs to be done the first time you load an instrument.
Afterward, pysat will identify them using the `platform` and `name` keywords.

```

> import pysat

Reviewer: `import pysatSpaceWeather`

Author: Actually, following the example above we don't need any more imports :P

Author: Oh, I see.  Registry needs the module imported.

> import pysat

Author: ```suggestion
```
